display_list/display_list_builder.cc

```diff
@@ -413,12 +414,29 @@ void DisplayListBuilder::setAttributesFromPaint(
   }
 }
 
-void DisplayListBuilder::save() {
+void DisplayListBuilder::checkForDeferredSave() {
```

Comment:
Perhaps put this in the .h file to encourage inlining since it is just a check around a method call?